### PR TITLE
Move help text in forms

### PR DESCRIPTION
**Pull request by Robert-Costello**
## Summary
[USH-497](https://dimagi-dev.atlassian.net/browse/USH-497)
Help text spans moved from below to above related select and input elements. This stops screen readers from reading headings multiple times and reads help text before the user enters input/select area as per accessibility guidelines. 

## Product Description
Help text appears above related input/select and is read before entering input/select area. 

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan
https://dimagi-dev.atlassian.net/browse/QA-3229

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by stephherbers** [comment]
Hey, @Robert-Costello just a few questions:
1. Did you see an issue where the type of element (i.e. text entry) is still being read after the actual element? I think it's great to also read the help text before, but this may not address the issue of the actual ticket. 
2. Do we know how reliable the help text is for teams? I believe the teams enter the help text themselves (please correct me if that's mistaken) so it may include not the right information (ex. "this is a text entry box")

Moving the elements should work in this case, but I believe accessibility best practice is using a aria-label for reading text that is not visible on the screen (this may be what is required if more than just the help-text needs modification): [https://www.aditus.io/aria/aria-label/](https://www.aditus.io/aria/aria-label/)

**Comment by stephherbers** [comment]
@Robert-Costello great! No need to add the aria-labels if that's not the issue

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less [hunk 1]
```diff
@@ -8,10 +8,6 @@
   display: block;
 }
 
-.sr-only {
-  display: none;
-}
-
```
Comment (stephherbers) [inline corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less:14]: Were you able to tell why this was added in the first place?

**Review by stephherbers — APPROVED** [review]
Looks good 👍

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less (hunk 1 above)
Comment (Robert-Costello) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less:14]: I checked all the places it was being used in this way, but it didn't seem to be critical at all. Everything that was completely hidden by this class seemed to function as it should with the bootstrap class. @biyeun made the change 6 years ago. Biyeun do you remember this?

Comment (biyeun) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less:14]: i'm sure it was important at the time. But 6 years ago...i definitely don't remember

**Pull request merged**